package/MDAnalysis/guesser/default_guesser.py

```diff
@@ -0,0 +1,606 @@
+# -*- Mode: python; tab-width: 4; indent-tabs-mode:nil; coding: utf-8 -*-
+# vim: tabstop=4 expandtab shiftwidth=4 softtabstop=4
+#
+# MDAnalysis --- https://www.mdanalysis.org
+# Copyright (c) 2006-2017 The MDAnalysis Development Team and contributors
+# (see the file AUTHORS for the full list of names)
+#
+# Released under the Lesser GNU Public Licence, v2 or any higher version
+#
+# Please cite your use of MDAnalysis in published work:
+#
+# R. J. Gowers, M. Linke, J. Barnoud, T. J. E. Reddy, M. N. Melo, S. L. Seyler,
+# D. L. Dotson, J. Domanski, S. Buchoux, I. M. Kenney, and O. Beckstein.
+# MDAnalysis: A Python package for the rapid analysis of molecular dynamics
+# simulations. In S. Benthall and S. Rostrup editors, Proceedings of the 15th
+# Python in Science Conference, pages 102-109, Austin, TX, 2016. SciPy.
+# doi: 10.25080/majora-629e541a-00e
+#
+# N. Michaud-Agrawal, E. J. Denning, T. B. Woolf, and O. Beckstein.
+# MDAnalysis: A Toolkit for the Analysis of Molecular Dynamics Simulations.
+# J. Comput. Chem. 32 (2011), 2319--2327, doi:10.1002/jcc.21787
+
+r"""
+Default Guesser
+================
+.. _DefaultGuesser:
+
+DefaultGuesser is a generic guesser class that has basic guessing methods.
+This class is a general purpose guesser that can be used with most topologies,
+but being generic makes it the less accurate among all guessers.
+
+
+
+
+
+Classes
+-------
+
+.. autoclass:: DefaultGuesser
+   :members:
+   :inherited-members:
+
+"""
+from .base import GuesserBase
+import numpy as np
+import warnings
+import math
+
+import re
+
+from ..lib import distances
+from . import tables
+
+
+class DefaultGuesser(GuesserBase):
+    """
+    This guesser holds generic methods (not directed to specific contexts) for
+    guessing different topology attribute. It has the same methods which where
+    originally found in Topology.guesser.py. The attributes that can be
+    guessed by this class are:
+    * masses
+    * types
+    * elements
+    * angles
+    * dihedrals
+    * bonds
+    * improper dihedrals
+    * aromaticities
```

Comment:
```suggestion
     - masses
     - types
     - elements
     - angles
     - dihedrals
     - bonds
     - improper dihedrals
     - aromaticities
```
Currently this isn't showing up as a list but rather "The attributes that can be guessed by this class are: * masses * types * elements * angles * dihedrals * bonds * improper dihedrals * aromaticities" in a single sentence. I _think_ this formatting change fixes this, but please double check.